src/SFML/Graphics/Text.cpp

```diff
@@ -453,7 +453,7 @@ void Text::ensureGeometryUpdate() const
         Uint32 curChar = m_string[i];
 
         // Skip the \r char to avoid weird graphical issues
-        if (curChar == '\r')
```

Comment:
Probably stupid question, but wouldn't uppercase U be more correct here? Considering it's a 32bit character type

Reply:
@Baardi you could be right, I don't know much about utf32. I would guess it's not needed, as the only characters in question are special characters ('\n', '\t', '\r' etc) and their bit structure (probably) won't change from 16 to 32 bits.

Reply:
We can't in SFML 2.6.x